service/dps/rest/src/main/java/eu/europeana/cloud/service/dps/rest/TopologyTasksResource.java

```diff
@@ -225,41 +219,51 @@ public Response submitTask(@Suspended final AsyncResponse asyncResponse,
         return Response.notModified().build();
     }
 
-
     /**
      * Retrieves notifications for the specified task.
      * <p/>
      * <br/><br/>
      * <div style='border-left: solid 5px #999999; border-radius: 10px; padding: 6px;'>
-         * <strong>Required permissions:</strong>
-         * <ul>
+        * <strong>Required permissions:</strong>
+        * <ul>
             * <li>Authenticated user</li>
             * <li>Read permission for selected task</li>
-         * </ul>
+        * </ul>
      * </div>
      *
      * @param taskId <strong>REQUIRED</strong> Unique id that identifies the task.
+     *               from : startingResourcesNum
+     *               to :   endingResourcesNum
      * @return Notification messages for the specified task.
      * @summary Retrieve task notifications
      */
     @GET
     @Path("{taskId}/notification")
     @PreAuthorize("hasPermission(#taskId,'" + TASK_PREFIX + "', read)")
```

Comment:
Missing produce content type (default is plain text and it can't produce xml). It could be on separate issue because this is related to whole dps app.